src/hardware/motor.cpp

```diff
@@ -92,6 +92,7 @@ float StepperMotor::getEstimRPM() {
 float StepperMotor::getDegreesPS() {
     calc:
     while (isStepping)
+        ; // wait end of stepping calculation
```

Comment:
Why put the comma on this line? Can't it be put on the same line as the while? Just minor preference for clean files

Reply:
Let use auto cede formatting.   #47